### PR TITLE
[GHA] [DOCS] New tests document

### PR DESCRIPTION
**Pull request by akashchi**
### Details:
Created upon #22102, please review only the new tests doc in this one.

### Tickets:
 - *128649*

### CODE REVIEW

**Comment by dorloff** [comment]
We need considerations on the expected time of tests execution - isn't this document an appropriate place?

File: docs/dev/ci/github_actions/adding_tests.md [hunk 1]
```diff
@@ -1,9 +1,117 @@
...
 
-### Add to the Already Existing Job
+The workflows have many jobs dedicated to building and testing of OpenVINO. This document describes the topic of adding 
+tests to these workflows or adding an entirely new workflow.
+
+## Add Tests to the Already Existing Workflow
+
+### Add Tests to the Existing Test Suit 
+
+If the new tests could be executed as a part of the already existing test suit, e.g., new OVC Python API tests,
```
Comment (dorloff) [inline docs/dev/ci/github_actions/adding_tests.md:12]: test suite?

Comment (dorloff) [inline docs/dev/ci/github_actions/adding_tests.md:12]: test suite?

**Comment by akashchi** [comment]
> We need to document expectations on the time of tests execution - isn't this document an appropriate place?

Good, I will add a subsection for it.

**Review by ilya-lavrenov — APPROVED** [review]

**Review by dorloff — APPROVED** [review]

**Pull request merged**